pages/common/s.md

```diff
@@ -0,0 +1,19 @@
+# s
+
+> Web search from the terminal.
+
+- Search for query on Google(default provider):
+
+`s {{query}}`
+
+- Search for a query on provider:
+
+`s --provider {{provider}} {{query}}`
+
+- List all providers:
+
+`s --list-providers`
+
+- Search for query in binary:
```

Comment:
Perhaps - `Use a specified binary to perform the search query` fits better. Also are the quotes necessary ? I believe it just needs the path to the binary file.

Reply:
The quotes aren't really necesseray, but mostly you would do something like: `chromium --incognito` and to get the arguments passed to `the binary` it is necessary, otherwise they would be interpreted by `s`. I can remove than or change them to `"{{binary}} {{arguments}}"`?

Reply:
Yes the latter is better.